docs/source/tutorials/first-steps.rst

```diff
@@ -0,0 +1,129 @@
+.. _tutorial-first-steps:
+
+First steps with webpush-channels API
+#####################################
+
+This tutorial will take you through your first API calls with a real
+webpush-channels server.
+
+In order to get the most out of this tutorial, you may want to have a
+real webpush-channels server ready. You can read our :ref:`installation
+<install>` guide to see how to set up your own webpush-channels instance if you
+like. We'll be using the :ref:`Mozilla demo server <run-webpush-channels-mozilla-demo>`.
+
+.. important::
+
+    In this tutorial we will use a Basic Authentication, which computes a
+    user id based on the token provided in the request.
+
+    This method has many limitations but has the advantage of not needing
+    specific setup or third-party services before you get started.
+
+    :ref:`Read more about authentication in webpush-channels <authentication>`.
+
+subscription and channel APIs
+=============================
+
+Using the `httpie <http://httpie.org>`_ tool we can post a sample subscription:
+
+.. note::
+
+    Please `consider reading httpie documentation <https://github.com/jkbrzt/httpie#proxies>`_
+    for more information (if you need to configure a proxy, for instance).
+
+.. code-block:: shell
+
+    $ echo '{"data": {"endpoint": "https://updates.push.services.mozilla.com/wpush/v1/gAAAAABYZNChoTLTAeA9vv-_zeqGuZiM4ESpiV7oiT5XtrN8aI01fiCQ7-_hC8lhqXanjUEWp5MFRoq35QmzdplCkRhp5nRgjwneGCGO8WXYH9psZaD_xInKLWm7K8-tzFAp-vRNHx79","keys": {"auth": "pnipzxpMvKBNYZAcxc-MAA","p256dh": "BEVoH6cOlNPuvYR0aVJo4GVv84nbymzpXxNff7hpKYjVIFcuIEtqiLtIe4rLOXF_A2w3KWRJoCYJEjUedrXcNpc"}}}' | http POST https://webpush-channels.dev.mozaws.net/v0/subscriptions -v --auth 'token:my-secret'
```

Comment:
Please describe how you grab the content of this. You can point people to https://ipsha21.github.io/service-worker-app/